src/Plate/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { range, uniq } from 'lodash';
+import React, { ReactNode } from 'react';
+
+import { MLL_THEME } from '../theme';
+
+export type PlateProps = {
+  data?: Array<{
```

Comment:
I would tend to extract and export the type within the `Array<>` as something like `Plate, I imagine it could be useful, e.g.:

```tsx
function preparePlateWells(apiResult): Array<PlateWell>
```